website/docs/r/org_vdc.html.markdown

```diff
@@ -88,6 +88,8 @@ The following arguments are supported:
 * `network_pool_name` - (Optional, System Admin) Reference to a network pool in the Provider VDC. Required if this VDC will contain routed or isolated networks.
 * `allow_over_commit` - (Optional) Set to false to disallow creation of the VDC if the AllocationModel is AllocationPool or ReservationPool and the ComputeCapacity you specified is greater than what the backing Provider VDC can supply. Default is true.
 * `enable_vm_discovery` - (Optional) If true, discovery of vCenter VMs is enabled for resource pools backing this VDC. If false, discovery is disabled. If left unspecified, the actual behaviour depends on enablement at the organization level and at the system level.
+* `elasticity` - (Optional, *v2.7+*) Indicates if the Flex vDC is to be elastic. Works only with Flex vCD.
+* `include_vm_memory_overhead` - (Optional, *v2.7+*) Indicate if the Flex vDC is to include memory overhead into its accounting for admission control. Works only with Flex vCD.
```

Comment:
@Didainius wrote:

> One general notice. As we are making a precedent with version dependent behavior - I think we need to decide on documentation.
I would vote of having something like 9.7+ for these fields in docs - both in website, and in the schema description.

I agree. What about this form:

```suggestion
* `include_vm_memory_overhead` - (Optional, *v2.7+*, *vCD 9.7+*) Indicate if the Flex vDC is to include memory overhead into its accounting for admission control. Works only with Flex vCD.
```

Reply:
changed